device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/hwsku.json

```diff
@@ -0,0 +1,212 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "default_brkout_mode": "1x1G",
+            "port_type": "RJ45"
```

Comment:
This attribute failed to pass this check during build: https://github.com/Azure/sonic-buildimage/blob/e5c2937faec615367a04075ab1a4dc7aeeee1be2/src/sonic-device-data/tests/hwsku_json_checker